src/config.rs

```diff
@@ -352,7 +355,7 @@ impl General {
     }
 
     pub fn default_idle_timeout() -> u64 {
-        60000 // 1 minute
+        600000 // 10 minutes
```

Comment:
Let's not change defaults unnecessarily. People are getting used to them, and it's much easier to change the config when a change is needed.

Reply:
That's a valid point.

I'm mainly changing this to match the pgbouncer default, but if we want to keep it as 1 minute then I can revert